src/components/editor/property/PropertyComponent.jsx

```diff
@@ -46,6 +46,7 @@ export class PropertyComponent extends Component {
         switch (property.type) {
           case 'literal':
             return (<InputLiteral key={keyId} id={keyId}
+                                  propertyTemplate = {property}
```

Comment:
i was tempted to rename the `property` parameter (in `inputComponentType`) to `propertyTemplate`, since that seemed like it'd be more accurate afaict.  but i wasn't sure about that.  if that does sound agreeable, i'm happy to add a touchup commit to this PR, or put up a tiny follow-on PR to that effect.

Reply:
👍 I think `propertyTemplate` reduces the ambiguity that the generic `property` parameter can introduce..